### PR TITLE
Draft: Production setup Doc

### PR DESCRIPTION
**Pull request by Fifciu**
Working on Production Setup Doc

### CODE REVIEW

File: docs/guide/installation/production-setup.md [hunk 1]
```diff
@@ -246,39 +246,50 @@ You need to clone the `vue-storefront` and the `vue-storefront-api` repos accord
...
+git clone https://github.com/DivanteLtd/vue-storefront-api.git;
+```
+
+Currently, theme is not part of PWA's repository anymore - you have to create `src/themes` and clone your theme inside:
+```bash
+cd /home/www/vuestorefront/src;
+mkdir themes;
+cd themes;
+# Clone one of these 2, default:
+git clone https://github.com/DivanteLtd/vsf-default.git;
```
Comment (gibkigonzo) [inline docs/guide/installation/production-setup.md:260]: If we don't want to change theme name in config then we should clone it to 'default' folder
`git clone https://github.com/DivanteLtd/vsf-default.git default;`
There is also info in docs about theme installation https://docs.vuestorefront.io/guide/installation/theme.html

File: docs/guide/installation/production-setup.md [hunk 2]
```diff
@@ -246,39 +246,50 @@ You need to clone the `vue-storefront` and the `vue-storefront-api` repos accord
...
+# or capybara theme
+git clone https://github.com/DivanteLtd/vsf-capybara.git;
```
Comment (gibkigonzo) [inline docs/guide/installation/production-setup.md:262]: `git clone https://github.com/DivanteLtd/vsf-capybara.git capybara;`
Capybara also has config specific for this theme. More info pt.4 https://github.com/DivanteLtd/vsf-capybara

File: docs/guide/installation/production-setup.md [hunk 3]
```diff
@@ -312,7 +323,7 @@ Please find the key sections of the `vue-storefront/config/local.json` file desc
 },
 ```
 
-We're setting up the product's endpoint to https://prod.vuestorefront.io/api/catalog (please use your domain accordingly of course). As you may notice, the `/api` url is proxied by the NGINX to `localhost:8080` - our `vue-storefront-api` instance.
+We're setting up the product's endpoint to https://prod.vuestorefront.io/api/catalog (please use your domain accordingly of course). As you may notice, the `https://prod.vuestorefront.io/api` url is proxied by the NGINX to `localhost:8080` - our `vue-storefront-api` instance.
```
Comment (gibkigonzo) [inline docs/guide/installation/production-setup.md:326]: We're setting up the product's endpoint => We're setting up the catalog endpoint

File: docs/guide/installation/production-setup.md [hunk 4]
```diff
@@ -332,13 +343,22 @@ We're setting up the product's endpoint to https://prod.vuestorefront.io/api/cat
   },
 ```
 
-There are 27 more instances of `prod.vuestorefront.io` to be replaced with your production URL address in this file. Please just do so :)
+There are many more instances of `https://prod.vuestorefront.io/api/*` to be replaced with your production URL address in this file. Please just do so :)
```
Comment (gibkigonzo) [inline docs/guide/installation/production-setup.md:346]: I'm not sure about this information :thinking: Right now also works relative path. So we don't need to add everywhere `https://prod.vuestorefront.io`

**Comment by Fifciu** [comment]
Add info about robots header from this file: https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/installation/etc/nginx/sites-enabled/prod.vuestorefront.io

**Comment by bloodf** [comment]
This is closed due to inactivity.

If you want your PR to be re-opened, please update your files, check for any conflicts.

If your PR is for Vue Storefront 1 target the new repository at https://github.com/vuestorefront/vue-storefront-1

**Pull request closed without merging**